### DISCUSSION TITLE
technology/learn/webapck2vite

### DISCUSSION THREAD

**Discussion by giscus** [ATQQ/sugar-blog#179] [Announcements]
# technology/learn/webapck2vite

粥里有勺糖的个人博客，记录随笔与学习笔记，大前端相关的知识，高频面试题，个人面经等

https://sugarat.top/technology/learn/webapck2vite.html

**Comment by simple5960** [comment]
很棒的文章，给了我启发，明天就去试试

**Reply by ATQQ** [reply]
思路应该可以借鉴但现在2024了实现的许多细节位置有一些变化😁
